app/main.cjsx

```diff
@@ -47,6 +47,7 @@ routes = <Route handler={App}>
     <DefaultRoute name="user-profile" handler={require './pages/profile/feed'} />
     <Route name="user-profile-private-message" path="message" handler={require './pages/profile/private-message'} />
     <Route name="user-profile-stats" path="stats" handler={require './pages/profile/stats'} />
+    <Route name="moderations" handler={require './talk/moderations'} />
```

Comment:
How about #/talk/moderation? Unless I'm misunderstanding: Is this only supposed to show a list of that user's comments that've been reported? I'm seeing all reported comments.